pandas/_libs/tslibs/dtypes.pyx

```diff
@@ -127,6 +131,7 @@ _attrname_to_abbrevs = {
     "nanosecond": "N",
 }
 cdef dict attrname_to_abbrevs = _attrname_to_abbrevs
+cdef dict _abbrev_to_attrnames = {v: k for k, v in attrname_to_abbrevs.items()}
```

Comment:
should make these both private or public

Reply:
followon ok

Reply:
the public one is used outside this module, the private one is not